Makefile

```
@@ -15,6 +15,13 @@
 # If you update this file, please follow
 # https://suva.sh/posts/well-documented-makefiles
 
+GO111MODULE = on
+export GO111MODULE
+GOFLAGS += -mod=vendor
```

This can cause a double add of the mod flag which causes issue
```suggestion
GOFLAGS ?= -mod=vendor
```